### PR TITLE
Setjob command

### PR DESCRIPTION
**Pull request by ggggg**
The jobs:
        Citizen
        Criminal
        Prisoner
        Police
        Paramedic
        Firefighter
        GangsterRed
        GangsterGreen
        GangsterBlue
        Mayor
        DeliveryDriver
        TaxiDriver
        SpecOps

### CODE REVIEW

File: dist/settings.json [hunk 1]
```diff
@@ -379,6 +379,12 @@
       "Commands": [
         "mute"
       ],
+    },
+    {
+      "CommandName": "SetJob",
+      "Commands": [
+        "setjob"
+      ],
```
Comment (PLASMAchicken) [inline dist/settings.json]:
```suggestion
      ]
```
Remove the trailing comma

File: src/BPEssentials/Commands/Player/SetJob.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public class SetJob : Command
+    {
+        public void Invoke(ShPlayer player, JobIndex job, ShPlayer target = null)
+        {
+            
+            target = target ?? player;
+            target.svPlayer.SvTrySetJob(job, true, false);
+            player.TS("job_set", target.username.CleanerMessage(), job.ToString());
+            if (target != player)
+                target.TS("new_job", player.username.CleanerMessage(), job.ToString());
```
Comment (PLASMAchicken) [inline src/BPEssentials/Commands/Player/SetJob.cs]:
```suggestion
            if (target != player) 
            {
                target.TS("new_job", player.username.CleanerMessage(), job.ToString());
            }
```
While not technically incorrect, the omission of curly braces can be misleading, and may lead to the introduction of errors during maintenance.

File: dist/settings.json [hunk 3]
```diff
@@ -378,6 +378,12 @@
       "CommandName": "ToggleMute",
       "Commands": [
         "mute"
+      ],
```
Comment (PLASMAchicken) [inline dist/settings.json]:
```suggestion
      ]
```
Remove trailing comma

File: dist/settings.json (hunk 1 above)
Comment (ggggg) [reply dist/settings.json]: already fixed

File: src/BPEssentials/Commands/Player/SetJob.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            
+            target = target ?? player;
+            target.svPlayer.SvTrySetJob(job, true, false);
+            player.TS("job_set", target.username.CleanerMessage(), job.ToString());
+            if (target != player) 
+            {
+                target.TS("new_job", player.username.CleanerMessage(), job.ToString());
+            }
+        }
+
```
Comment (PLASMAchicken) [inline src/BPEssentials/Commands/Player/SetJob.cs]:
```suggestion
```

**Pull request merged**